### DISCUSSION TITLE
FYI: sokol-shdc regression fix

### DISCUSSION THREAD

**Discussion by floooh** [kassane/sokol-d#35] [General]
Hi @kassane, please be aware of this fix in sokol-shdc, might want to update the sokol-shdc dep in build.zig.zon:

https://github.com/floooh/sokol-tools/blob/master/CHANGELOG.md#09-nov-2024

**Comment by kassane** [comment]
Thanks for clarifying.

**Comment by floooh** [comment]
Btw another one after the first fix - might want to keep an eye on this issue until the dust has settled ;)

https://github.com/floooh/sokol-tools/issues/154
